zadani/soft/style.css

```diff
@@ -1,43 +1,304 @@
+                                                                                                                    font-size: 10px;
+                                                                                                                }
+                                                                                                                
+                                                                                                                .zdroje {
+                                                                                                                    padding-top: 40px;
+                                                                                                                    margin: 0;
+                                                                                                                }
+                                                                                                                
+                                                                                                                footer a {
+                                                                                                                    color: rgb(153, 149, 149);
+                                                                                                                }
+                                                                                                                /*tablet*/
+                                                                                                                
+                                                                                                                @media (min-width: 500px) {
+                                                                                                                    /* MENU */
+                                                                                                                    .menu {
+                                                                                                                        display: flex;
+                                                                                                                        justify-content: space-around;
+                                                                                                                        border-top: 1px rgb(228, 227, 227) solid;
+                                                                                                                        border-bottom: 1px rgb(228, 227, 227) solid;
+                                                                                                                        padding: 0;
+                                                                                                                    }
+                                                                                                                    .menu li {
```

Comment:
Tady by bylo asi nejjednodušší nastavit float a pevnou šířku, než to komplikovat flexboxem:
    float: left;
    width: 25%;

Reply:
To vyřeší i problém se zarovnáním borderu